internal/topic/topicreaderinternal/stream_reconnector_test.go

```diff
@@ -256,7 +262,10 @@ func TestTopicReaderReconnectorConnectionLoop(t *testing.T) {
 	t.Run("StartWithCancelledContext", func(t *testing.T) {
 		ctx, cancel := xcontext.WithCancel(context.Background())
 		cancel()
-		reconnector := &readerReconnector{tracer: &trace.Topic{}}
+		reconnector := &readerReconnector{
+			tracer:     &trace.Topic{},
+			initDoneCh: make(empty.Chan),
```

Comment:
the chan must be inited with initChannelsAndClock

Reply:
in this case initChannelsAndClock() shouldn't be used, because other channels won't be closed